lib/trento_web/controllers/v1/activity_log_json.ex

```diff
@@ -41,6 +42,12 @@ defmodule TrentoWeb.V1.ActivityLogJSON do
     }
   end
 
+  defp map_severity_integer_to_text(n) when n >= 5 and n <= 8, do: "debug"
+  defp map_severity_integer_to_text(n) when n >= 9 and n <= 12, do: "info"
+  defp map_severity_integer_to_text(n) when n >= 13 and n <= 16, do: "warning"
+  defp map_severity_integer_to_text(n) when n >= 17 and n <= 20, do: "error"
+  defp map_severity_integer_to_text(n) when n >= 21 and n <= 24, do: "critical"
+
```

Comment:
These fit well in `Trento.ActivityLog.SeverityLevel` so we have bi directional translation in one place.